zstacklib/zstacklib/utils/sanlock.py

```
@@ -515,3 +663,76 @@ def get_watchdog_fire_timeout():
+        return None
+
+    from zstacklib.utils import lvm
+    if resource_name == "VGLK":
+        # send repair metadata request to mn
+        lease = PaxosLease()
+        lease.lockspace_name = lockspace_name
+        lease.resource_name = resource_name
+        lease.sector_size = get_sector_size(vg_name)
+        lease.align_size = sector_size_to_align_size(lease.sector_size)
+        lease.offset = VGLK_BEGIN * lease.align_size
+        return request_mn_repair_metadata(vg_name, **lease.__dict__)
```

Pylint (3.3.7)

[error] Argument 'vg_name' passed by position and keyword in function call

(E1124)